gym/envs/__init__.py

```diff
@@ -774,9 +773,9 @@ def _merge(a, b):
 
         register(
             id="{}-v0".format(name),
-            entry_point="gym.envs.atari:AtariEnv",
+            entry_point="ale_py.gym:ALGymEnv",
             kwargs={
-                "game": game,
+                "game": name,
```

Comment:
The environment now takes the game name in camel case instead of the ID in snake case. This kwarg was only used by Gym as `atari-py` was what interpreted the game ID. From what I can tell no one else uses this kwarg.

Reply:
This has now been reverted to `game`.